syntaxes/omg.idl.tmLanguage.json

```diff
@@ -142,7 +142,8 @@
         "type_dcl": {
             "patterns": [
                 {"include": "#constr_type_dcl"},
-                {"include": "#typedef_dcl"}
+                {"include": "#typedef_dcl"},
+                {"include": "#interface_def"}
             ]
```

Comment:
I think we need to add also the ability to declare exceptions.

Reply:
Done.